### PR TITLE
improvement(latte): improve latte errors parsing

### PR DESCRIPTION
**Pull request by vponomaryov**
Current code prints first 2 lines of the stderr when a latte stress command fails.
It is always useless lines because latte writes a lot of stuff into the stderr.

So, change error parsing the following way:
- print out "last" 2 lines in a common case with failures
- print out full "panic" message if it appears.

### Testing

- [scylla-staging/valerii/vp-longevity-gce-custom-d1-workload1-hybrid-raid#25](https://argus.scylladb.com/test/2f38fb93-05bb-4949-8822-31707d5d9e76/runs?additionalRuns[]=ba8feab1-50ef-41b8-b0a2-dc93157bba02)

### PR pre-checks (self review)

- [x] I added the relevant `backport` labels
- [x] I didn't leave commented-out/debugging code

### Reminders

- Add New configuration option and document them (in `sdcm/sct_config.py`)
- Add unit tests to cover my changes (under `unit-test/` folder)
- Update the Readme/doc folder relevant to this change (if needed)

### CODE REVIEW

**Comment by vponomaryov** [comment]
The `latte panic` example is here: https://github.com/scylladb/qa-tasks/issues/1796

File: sdcm/stress/latte_thread.py [hunk 1]
```diff
@@ -227,3 +229,29 @@ def _run_stress(self, loader, loader_idx, cpu_idx):
         return {}
         # TODOs:
         # 1) take back the report workload..3.0.8.p128.t1.c1.20231025.220812.json
+
+    def configure_event_on_failure(self, stress_event: LatteStressEvent, exc: Exception | Failure):
```
Comment (soyacz) [inline sdcm/stress/latte_thread.py:233]: It is the same as in `DockerBasedStressThread` - is it needed?

Comment (vponomaryov) [reply sdcm/stress/latte_thread.py:233]: It looks the same, but without it's explicit definition here we cannot use the custom `format_stress_cmd_error` function.

Comment (soyacz) [reply sdcm/stress/latte_thread.py:233]: uh, now I see `format_stress_cmd_error` is not part of the object.

**Review by soyacz — APPROVED** [review]
LGTM

**Review by fruch — APPROVED** [review]
LGTM

**Pull request merged**